src/Compilers/CSharp/Test/Semantic/FlowAnalysis/RegionAnalysisTests.cs

```diff
@@ -6360,6 +6363,41 @@ void Method(out object test)
             Assert.Equal("this, test, a", GetSymbolNamesJoined(dataFlowAnalysisResults.WrittenOutside));
         }
 
+        [Fact, WorkItem(37427, "https://github.com/dotnet/roslyn/issues/37427")]
+        public void RegionWithLocalFunctions()
+        {
+            var analysisResults = CompileAndAnalyzeDataFlowStatements(@"
+class A
+{
+    static void M(int p)
+    {
+        int i, j;
+        i = 1;
+        /*<bind>*/
+        int L1() => 1;
```

Comment:
We should have a test that confirms that this code behaves identically if the local function declarations are moved out of the region (I think? That's my understanding of the contract of these region analyses).

Reply:
That will be true for this test since the local functions are trivial.  I will add that to this test.